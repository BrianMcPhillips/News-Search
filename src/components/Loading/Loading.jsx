import React from 'react';
import styles from './Loading.css';

const Loading = () => (
  <img 
    className={styles.Loading}
    src="https://media0.giphy.com/media/j247L1e4KaY6bEBZQ0/giphy.gif" 
    alt="newspaper loading screen"/>
);

export default Loading;
