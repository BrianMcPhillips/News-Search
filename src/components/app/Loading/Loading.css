.Loading {
  display: block;
  margin: auto;
  max-width: 15rem;
}