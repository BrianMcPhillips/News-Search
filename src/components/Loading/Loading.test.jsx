import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Loading from './Loading';

describe('Loading screen component', () => {
  afterEach(() => cleanup());
  it('displays the loading image', () => {
    const { asFragment } = render(<Loading />);
    
    expect(asFragment()).toMatchSnapshot();
  });
});
