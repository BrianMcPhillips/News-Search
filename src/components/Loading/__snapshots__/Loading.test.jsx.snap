// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading screen component displays the loading image 1`] = `
<DocumentFragment>
  <img
    alt="newspaper loading screen"
    class="Loading"
    src="https://media0.giphy.com/media/j247L1e4KaY6bEBZQ0/giphy.gif"
  />
</DocumentFragment>
`;
